### PR TITLE
Add LoggingBackend; Add headers; Make json payload not a string; default configuration

### PR DESCRIPTION
**Pull request by julienledem**

### CODE REVIEW

File: src/test/java/marquez/client/MarquezHttpTest.java [hunk 1]
```diff
@@ -223,4 +227,14 @@ public void testGet_throwsOnHttpError() throws Exception {
     final URL url = marquezUrl.from(path("/namespace/%s", newNamespaceName()));
     assertThatExceptionOfType(MarquezHttpException.class).isThrownBy(() -> marquezHttp.get(url));
   }
+
+  @Test
+  public void testClose() throws Exception {
+    // the sclient is not closeable
+    marquezHttp.close();
```
Comment (wslulciuc) [inline src/test/java/marquez/client/MarquezHttpTest.java:234]: _Minor:_ Does this line affect the test, or is it more for documentation?

**Review by wslulciuc — APPROVED** [review]
Minor comment, otherwise LGTM 👍

File: src/test/java/marquez/client/MarquezHttpTest.java (hunk 1 above)
Comment (julienledem) [reply src/test/java/marquez/client/MarquezHttpTest.java:234]: it does cover that close() is  a no op when the client is not closeable. Notably it doesn't throw an exception.

Comment (wslulciuc) [reply src/test/java/marquez/client/MarquezHttpTest.java:234]: I see. We could move the assertion into its own test if we wanted to make it more explicit, but either way works 👍

**Pull request merged**